### ISSUE TITLE
dead object error in ProcessSidebar

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#841]
_[Trac time 20100513 195156Z]_ Using the Tools -> Run option after a peppy window has been closed results in:

```
Traceback (most recent call last):
  File "/opt/lib/python2.5/site-packages/wx-2.8-gtk2-unicode/wx/_core.py", line 14614, in <lambda>
    lambda event: event.callable(*event.args, **event.kw) )
  File "/data2/home/rob/src/peppy/peppy/lib/processmanager.py", line 473, in jobStartedCallback
    Publisher().sendMessage('peppy.processmanager.started', job)
  File "/opt/lib/python2.5/site-packages/wx-2.8-gtk2-unicode/wx/lib/pubsub.py", line 811, in sendMessage
    self.__topicTree.sendMessage(aTopic, message, onTopicNeverCreated)
  File "/opt/lib/python2.5/site-packages/wx-2.8-gtk2-unicode/wx/lib/pubsub.py", line 498, in sendMessage
    deliveryCount += node.sendMessage(message)
  File "/opt/lib/python2.5/site-packages/wx-2.8-gtk2-unicode/wx/lib/pubsub.py", line 336, in sendMessage
    listener(message)
  File "/data2/home/rob/src/peppy/peppy/lib/processmanager.py", line 541, in msgStarted
    self.reset()
  File "/opt/lib/python2.5/site-packages/wx-2.8-gtk2-unicode/wx/_core.py", line 14564, in __getattr__
    raise PyDeadObjectError(self.attrStr % self._name)
PyDeadObjectError: The C++ part of the ProcessSidebar object has been deleted, attribute access no longer allowed.
```

**Comment by robmcmullen** [comment]
_[Trac time 20100513 195316Z]_ Probably fixable by moving to pubsub3

**Comment by robmcmullen** [comment]
_[Trac time 20100517 235740Z]_ f5413a3be14607dd0b1b19d2b901eda6b3fa3208 Fixed #841: sidebar panes and springtab weren't getting closed properly and the process sidebar reference wasn't getting cleaned up
